### ISSUE TITLE
Update docu to avoid duplicated efforts

### ISSUE THREAD

**Issue by FlorianLiers** [AddonScriptorDE/plugin.video.ardmediathek_de#6]
I was not aware of your second repository (http://code.google.com/p/addonscriptorde-beta-repo/source/browse/#svn). Is there any hint to it in the GitHub repo? The XBMC web page links to GitHub. At least It would be nice to have some hint to it. With such a hint, I could have saved some time for fixing #2, #3, and #4 a second time.

Just out of curiosity: Why do you have two repos? Wouldn't it be easier to just have one in order to incorporate feedback from others? The versioning issue (marking a version as "V 2.x.y") could be solved by creating branches.

**Comment by AddonScriptorDE** [comment]
Yes, sorry. I will change this. I needed an own xbmc repo, from which people could install addons and just created one for testing at google. See reasons here: http://forum.xbmc.org/showthread.php?tid=156190 But then more and more addons/updates followed. When I have some time I will change it, so everything here on github will be up2date again, and people can install addons directly via xbmc from this repo. The main problem was, that the addons have to be in one subrepo to be installable via xbmc. Like here: https://github.com/Ruxton/xbmc-addon-repo 
Should be possible via submodules/subtrees. But I never used github before and dont know anything about it. Will have a look at it.
About the branches: It wouldnt solve the 2 versions. It was not for me, it was because of some internal xbmc eden/frodo problem: http://forum.xbmc.org/showthread.php?tid=173887&pid=1511985#pid1511985 I don't wanted to create branches because it was just temporary and I'm not supporting eden anymore...
